src/main/kotlin/com/hyperdevs/poeditor/gradle/Main.kt

```diff
@@ -31,6 +31,7 @@ fun main() {
 
     val apiToken = dotenv.get("API_TOKEN", "")
     val projectId = dotenv.get("PROJECT_ID", "-1").toInt()
+    val resFileName = dotenv.get("RES_FILE_NAME", "")
```

Comment:
I'd just set this to "strings"
```suggestion
    val resFileName = dotenv.get("RES_FILE_NAME", "strings")
```

Reply:
Done!